src/components/Pages/Collection/index.js

```diff
@@ -97,7 +97,7 @@ const Collection = ({ id, location }) => {
     let query = ''
     if (typeof generalPartiallyDigitized !== 'undefined') {
       query = `mutation {
-          replacePartiallyDigitized(input: {
+          savePartiallyDigitizedForWebsite(input: {
```

Comment:
Parameters changed here.  New parameters are:  itemId and partiallyDigitized.

Reply:
thanks didn't see that the inputs changed